### PR TITLE
Enable 3 node deployment

### PR DESCRIPTION
**Pull request by Jorres**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Other information

Migrating from 8-node YDB cluster with block-4-2 erasure to 3-node YDB cluster with mirror-3-dc erasure allowed us to run tests on top of Github runners, eliminating the need to supply secrets and a huge chunk of workflow YAML code. 

Todo @Jorres: after this merges, remove unnecessary secrets from this repo

### CODE REVIEW

**Comment by Jorres** [comment]
Cancelled tests are okay! The ones triggered by 'pull request' are the same as the one triggered by 'pull_request_target'. After this merges, the 'pull_request_target' will no longer exist and only one set of jobs, without `check-pr`, will run in the repository. 

For the proof that an outside person can not run the workflow without approval, see this closed PR, which I opened from another account, not belonging to YDB Platform: https://github.com/ydb-platform/ydb-kubernetes-operator/pull/250

File: samples/storage-mirror-3dc.yaml
Comment (kobzonega) [inline samples/storage-mirror-3dc.yaml:14]: Check this please. Maybe we should use the last version from website "Downloads" page https://ydb.tech/docs/en/downloads/?

File: .github/workflows/run-tests.yml
Comment (kobzonega) [inline .github/workflows/run-tests.yml:8]: I want to have ability to manual run workflow. Event type `workflow_dispatch` should be enabled for that
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow

**Review by kobzonega — APPROVED** [review]

**Pull request merged**
